data/pools/transactionPool.go

```diff
@@ -293,6 +293,16 @@ func (pool *TransactionPool) computeFeePerByte() uint64 {
 // checkSufficientFee take a set of signed transactions and verifies that each transaction has
 // sufficient fee to get into the transaction pool
 func (pool *TransactionPool) checkSufficientFee(txgroup []transactions.SignedTxn) error {
+	// Special case: the compact cert transaction, if issued from the
```

Comment:
Economy of mechanism:

- These certificates are likely to be generated by nodes that aren't block proposers, so they need to be broadcast and queued by nodes that eventually propose a block.

- There might be multiple certs that appear in a single block, and they might take a non-trivial amount of block space; the tx pool needs to take them into account.

- The certs are much larger than our current block header, so they should not literally appear in the block header.  Having them as a transaction gives the right properties: the header has a hash commitment to the payset (which includes the cert transaction), and the header doesn't become huge.

- It's convenient to be able to look them up by sender address, like other transactions.

It's entirely possible to implement another part of the block that looks much like the current transaction set, but instead is used for "system messages".  It would mostly duplicate the transaction machinery: its own message broadcast protocol, its own message pool, its own hash commitment in the header, its own lookup API, etc.